ve_faker:
  ros_parameters:
    mean_vx: 0.0
    mean_vy: 0.0
    mean_vyaw: 0.0
    stddev_vx: 0.1
    stddev_vy: 0.18
    stddev_vyaw: 0.15
    mean_ax: 0.0
    mean_ay: 0.0
    stddev_ax: 0.11
    stddev_ay: 0.2
